pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -126,19 +101,26 @@ func SetDefaults_ControllerManagerConfiguration(obj *ControllerManagerConfigurat
 	}
 }
 
+// SetDefaults_ClientConnection sets defaults for the client connection.
+func SetDefaults_ClientConnection(obj *apimachineryconfigv1alpha1.ClientConnectionConfiguration) {
+	apimachineryconfigv1alpha1.RecommendedDefaultClientConnectionConfiguration(obj)
+}
+
+// SetDefaults_GardenerClientConnection sets defaults for the client connection.
+func SetDefaults_GardenerClientConnection(obj *apimachineryconfigv1alpha1.ClientConnectionConfiguration) {
+	apimachineryconfigv1alpha1.RecommendedDefaultClientConnectionConfiguration(obj)
```

Comment:
This uses `application/vnd.kubernetes.protobuf`, but at this moment we don't support it. This can break the controller.